Bugzilla/CPAN.pm

```diff
@@ -104,6 +104,16 @@ sub feature {
     return $FEATURE_LOADED{$feature_name} = 1;
 }
 
+sub preload_features {
+    my ($class) = @_;
+    return 0 unless _CAN_HAS_FEATURE;
+    my $meta = $class->cpan_meta;
+
+    foreach my $feature ($meta->features) {
+        $class->feature($feature->identifier);
+    }
```

Comment:
This is more concise and with two less my() invocations, but the usual concerns about map() readability apply. ```map { $class->feature($_->identifier) } $class->cpan_meta->features;``` might work out.